postgres-appliance/convert_config.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import yaml
+
+keys_move = {
+    "bootstrap.dcs.ttl": "ttl",
+    "bootstrap.dcs.loop_wait": "loop_wait",
+    "bootstrap.dcs.maximum_lag_on_failover": "postgresql.maximum_lag_on_failover",
+    "bootstrap.dcs.postgresql.parameters": "postgresql.parameters",
+    "bootstrap.dcs.postgresql.recovery_conf": "postgresql.recovery_conf",
+    "bootstrap.initdb": "postgresql.initdb",
+    "bootstrap.users.admin.password": "postgresql.admin.password"
+}
+
+keys_delete = [
+    "postgresql.admin"
+]
+
+keys_add = {
+    "bootstrap.users.admin.options": ["createrole", "createdb"],
+    "bootstrap.dcs.postgresql.use_pg_rewind": True,
+    "bootstrap.dcs.postgresql.use_slots": True,
+    "bootstrap.dcs.retry_timeout": 10
+}
+
+keys_copy = {
+    "bootstrap.dcs.postgresql.authentication.replication": "postgresql.replication",
```

Comment:
superuser and replication keys are not taken from bootstrap section, but from the postgresql.authentication